doc/theoretical_description_regression.rst

```diff
@@ -241,6 +241,54 @@ Note that this means that using the split method will require to run three separ
 to estimate the prediction intervals.
 
 
+9. The ensemble batch prediction intervals (EnbPI) method
+=========================================================
+
+The coverage guarantee offered by the various resampling methods based on the
+jackknife strategy, implemented in MAPIE, are only valid under the "exchangeability
+hypothesis". It means that the probability law of data should not change up to
+reordering.
+This hypothesis is not revelant in many cases, notably for dynamical times series.
+That is why a specific class is needed, namely
+:class:`mapie.time_series_regression.MapieTimeSeriesRegressor`.
+
+Its implementation looks like the jackknife+-after-bootstrap method. The 
+LOO estimators are approximated thanks to a few boostraps. However the
+confidence intervals are like those of the jackknife method.
+
+.. math::
+  \hat{C}_{n, \alpha}^{\rm EnbPI}(X_{n+1}) = [\hat{\mu}_{agg}(X_{n+1}) + \hat{q}_{n, \beta}\{ R_i^{\rm LOO} \}, \hat{\mu}_{agg}(X_{n+1}) + \hat{q}_{n, (1 - \alpha + \beta)}\{ R_i^{\rm LOO} \}]
+where :math:`\hat{\mu}_{agg}(X_{n+1})` is the aggregation of the predictions of
+the LOO estimators (mean or median).
+
+The residuals are no longer considered in absolute values but in relative
+values and the width of the confidence intervals are minimized, up to a given gap
+between the quantiles' level.
```

Comment:
It would be worth introducing the $\beta$ parameter.

Reply:
Done